core/server/data/migration/fixtures/007/01-add-themes-permissions.js

```diff
@@ -0,0 +1,33 @@
+var utils = require('../utils'),
+    permissions = require('../../../../permissions'),
+    resource = 'theme';
+
+function getPermissions() {
+    return utils.findModelFixtures('Permission', {object_type: resource});
```

Comment:
Note: not sure if this is really a big problem, but this method assumes it's going to be adding all the permissions for a particular resource.
In this case we are only adding the 2 new theme permissions, so the messaging will look like:

> Skipping Migrations: (3/5) Adding permissions fixtures for themes
> Skipping Migrations: (3/5) Adding permissions_roles fixtures for themes